server/app.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const { pipeline } = require('stream');
+const zlib = require('zlib');
+const { StringDecoder } = require('node:string_decoder');
+const { multiparty } = require('multiparty');
+
+const server = http.createServer((req, res) => {
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  // res.setHeader('Access-Control-Allow-Credentials', 'true');
+  // res.setHeader('Access-Control-Allow-Methods', 'GET,HEAD,OPTIONS,POST,PUT');
+
+  res.setHeader('Access-Control-Allow-Headers',
+    'Access-Control-Allow-Headers, Origin, Accept, X-Requested-With,'
+      + 'Content-Type, Access-Control-Request-Method,'
+      + 'Access-Control-Request-Headers');
+  res.setHeader('Content-Disposition', 'attachment');
```

Comment:
Do you really need it while working locally?

Reply:
If I launch index.html by LiveServer, yes, I need it. But in new solution no))